### PR TITLE
Run pylint as separate GH action

### PR DESCRIPTION
**Pull request by crazyscientist**
... and allow lines to be 100 characters long.

This closes python-bugzilla#185

### CODE REVIEW

**Comment by crazyscientist** [comment]
@crobinso 

I noticed, that there is also `flake8` used in the other actions, but its "silenced". Can/Should I remove it?

File: bugzilla/base.py [hunk 1]
```diff
@@ -1489,7 +1489,7 @@ def s(key, val, convert=None):
                 val = convert(val)
             ret[key] = val
 
-        def add_dict(key, add, remove, _set=None, convert=None):
```
Comment (crazyscientist) [inline bugzilla/base.py:1492]: If I'm not mistaken, all uses of `convert` inside of the method were removed by #190. Now `pylint` complained about the argument not being used.

I checked with `grep` and this method never receives 5 arguments.

**Comment by crazyscientist** [comment]
Functional RW tests failed because staging server became unavailable:

```
$ pytest --rw-functional --ro-functional
[...]
=== short test summary info ===
FAILED tests/test_rw_functional.py::test0LoggedInNoCreds[REST] - bugzilla.exceptions.BugzillaHTTPError: 503 Server Error: Service Unavailable for url: https://bugzilla.stage.redhat.co...
FAILED tests/test_rw_functional.py::test04NewBugAllFields[REST] - bugzilla.exceptions.BugzillaHTTPError: 503 Server Error: Service Unavailable for url: https://bugzilla.stage.redhat.co...
FAILED tests/test_rw_functional.py::test05ModifyStatus[REST] - bugzilla.exceptions.BugzillaHTTPError: 503 Server Error: Service Unavailable for url: https://bugzilla.stage.redhat.co...
FAILED tests/test_rw_functional.py::test06ModifyEmails[REST] - bugzilla.exceptions.BugzillaHTTPError: 503 Server Error: Service Unavailable for url: https://bugzilla.stage.redhat.co...
FAILED tests/test_rw_functional.py::test070ModifyMultiFlags[REST] - bugzilla.exceptions.BugzillaHTTPError: 503 Server Error: Service Unavailable for url: https://bugzilla.stage.redhat.co...
FAILED tests/test_rw_functional.py::test071ModifyMisc[REST] - bugzilla.exceptions.BugzillaHTTPError: 503 Server Error: Service Unavailable for url: https://bugzilla.stage.redhat.co...
FAILED tests/test_rw_functional.py::test08Attachments[REST] - bugzilla.exceptions.BugzillaHTTPError: 503 Server Error: Service Unavailable for url: https://bugzilla.stage.redhat.co...
FAILED tests/test_rw_functional.py::test09Whiteboards[REST] - bugzilla.exceptions.BugzillaHTTPError: 503 Server Error: Service Unavailable for url: https://bugzilla.stage.redhat.co...
FAILED tests/test_rw_functional.py::test11UserUpdate[REST] - bugzilla.exceptions.BugzillaHTTPError: 503 Server Error: Service Unavailable for url: https://bugzilla.stage.redhat.co...
FAILED tests/test_rw_functional.py::test11ComponentEditing[REST] - bugzilla.exceptions.BugzillaHTTPError: 503 Server Error: Service Unavailable for url: https://bugzilla.stage.redhat.co...
FAILED tests/test_rw_functional.py::test13SubComponents[REST] - bugzilla.exceptions.BugzillaHTTPError: 503 Server Error: Service Unavailable for url: https://bugzilla.stage.redhat.co...
FAILED tests/test_rw_functional.py::test14ExternalTrackersAddUpdateRemoveQuery[REST] - bugzilla.exceptions.BugzillaHTTPError: 503 Server Error: Service Unavailable for url: https://bugzilla.stage.redhat.co...
FAILED tests/test_rw_functional.py::test15EnsureLoggedIn[REST] - bugzilla.exceptions.BugzillaHTTPError: 503 Server Error: Service Unavailable for url: https://bugzilla.stage.redhat.co...
FAILED tests/test_rw_functional.py::test16ModifyTags[REST] - bugzilla.exceptions.BugzillaHTTPError: 503 Server Error: Service Unavailable for url: https://bugzilla.stage.redhat.co...
FAILED tests/test_rw_functional.py::test17LoginAPIKey[REST] - bugzilla.exceptions.BugzillaHTTPError: 503 Server Error: Service Unavailable for url: https://bugzilla.stage.redhat.co...
FAILED tests/test_rw_functional.py::test0ClassDetection - bugzilla.exceptions.BugzillaHTTPError: 503 Server Error: Service Unavailable for url: https://bugzilla.stage.redhat.co...
FAILED tests/test_rw_functional.py::test10Login - AssertionError: assert 'Login failed: ' in '\nConnection lost/failed: 503 Server Error: Service Unavailable for url: h...
=== 17 failed, 129 passed in 281.34s (0:04:41) ===

**Comment by crazyscientist** [comment]
All tests passed :heavy_check_mark: 

```
$ pytest --rw-functional --ro-functional
=== 146 passed in 371.12s (0:06:11) ====

File: bugzilla/base.py (hunk 1 above)
Comment (crazyscientist) [reply bugzilla/base.py:1492]: Sure, makes sense :+1: . See #195.

**Review by crobinso — APPROVED** [review]

**Pull request merged**